docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+.. py:currentmodule:: datatable
+
+Comparison with Rdatatable
+==========================
+
+``Datatable`` strives to mimic `Rdatatable's <https://rdatatable.gitlab.io/data.table/index.html>`_ API as closely as possible; however, there are differences due to language constraints.
+
+This page shows how to perform similar basic operations in `Rdatatable <https://rdatatable.gitlab.io/data.table/index.html>`_  in ``datatable``.
+
+Subsetting Rows
+----------------
```

Comment:
We normally use as many underscores as there is characters in the title, here it seems that you have one extra. This is not a big deal though.